config/connections.js

```diff
@@ -24,9 +24,9 @@ const mongodb_connect = function () {
   * @name kafka_connect
 */
 
-const kafka_connect = function() {
-  global.kafkaClient = require("./kafka")();
-};
+//const kafka_connect = function() {
```

Comment:
@VISHNUDAS-tunerlabs please uncomment this.If you comment this it wont work in server